api/public_api_v1.yaml

```diff
@@ -4,40 +4,48 @@ info:
   title: Digital Citizenship API
   description: Digital Citizenship API.
 host: localhost
-basePath: /api/v1
+basePath: "/api/v1"
 schemes:
-  - https
+- https
 paths:
-  '/messages/{id}':
+  "/messages/{fiscal_code}/{id}":
     parameters:
-      - name: id
-        in: path
-        type: string
-        required: true
-        description: The ID of the message.
+    - name: fiscal_code
+      in: path
+      type: string
```

Comment:
is it possible to reference `#/definitions/FiscalCode` here?

Reply:
According to [OpenAPI 2.0 schema](https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v2.0/schema.json#L810) you cannot use "$ref" pointers to #/definitions/* with parameters but you can create a $ref pointer to #/parameters/*. Moreover these parameters defintions cannot contain $ref pointers to #/definitions so you end up to have two separate (but identical) definitions for FiscalCode in the best case. That's what I'm going to plug into the specs anyway.